src/main/java/seedu/address/model/event/DeadlineTable.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.event;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.TreeMap;
+
+import seedu.address.model.event.exceptions.EmptyEventException;
+
+public class DeadlineTable {
+    private final HashMap<String, DayDeadline> deadlineTableHash;
+
+    public DeadlineTable() {
+        this.deadlineTableHash = new HashMap<String, DayDeadline>();
+    }
+
+    public String add(Deadline deadline) {
+        if (this.deadlineTableHash.get(deadline.getDate()) == null) {
+            DayDeadline dayDeadline = new DayDeadline();
+            dayDeadline.addDeadline(deadline);
+            this.deadlineTableHash.put(deadline.getDate(), dayDeadline);
+            return "Task added to schedule";
+        }
+        return this.deadlineTableHash.get(deadline.getDate()).addDeadline(deadline);
+    }
+
+    public String delete(Deadline deadline) {
+        String UIMessage = this.deadlineTableHash.get(deadline.getDate()).deleteDeadline(deadline);
```

Comment:
Change to `UiMessage`, currently this has 3 consecutive capital letters, which violates the naming convention.